universal-application-tool-0.0.1/app/views/BaseHtmlView.java

```diff
@@ -78,4 +89,31 @@ protected String renderDateTime(Instant time) {
     LocalDateTime datetime = LocalDateTime.ofInstant(time, ZoneId.of("America/Los_Angeles"));
     return datetime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd 'at' h:mm a"));
   }
+
+  protected ImmutableList<DomContent> createLinksAndEscapeText(String content) {
```

Comment:
suggestion: Maybe move this to the `ProgramIndexView` class since it only seems relevant there - unless I'm misunderstanding its potential general use.

Reply:
We'll want it when we do the issue about being able to add text to the questionnaire.  or maybe in questions?